documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,233 @@
+
+[NOTE]
+====
+You need an installed Docker version for the https://quarkus.io/guides/datasource#dev-services[zero config setup].
+==== 
+
+
+.Database Migration
+. Migration guide
+For schema-based databases, we recommend migrating databases with Flyway.
+In that case there is our link:../guide-database-migration.asciidoc[Quarkus non-specific migration guide] can give you an overview if you are not familiar with migration.
+.. Flyway guide for Quarkus
+This Quarkus guide will show how to work with the https://quarkus.io/guides/flyway[Flyway extension in a Quarkus application].
+This should be used if you start your own database and do not leave the creation to quarkus.
+
+
+== Packaging of a Quarkus application and creation of a native Executable
+Quarkus applications can be packed into different types. The following link will show how to build and also give you a short explanation of the characteristics of these files.
+
+.Package types
+. https://quarkus.io/guides/maven-tooling#fast-jar[fast-jar]
+. https://quarkus.io/guides/maven-tooling#remote-development-mode[mutable-jar]
+. https://quarkus.io/guides/maven-tooling#uber-jar-maven[uber-jar]
+. link:guide-native-image.asciidoc[native executable]
+
+To pack an application use the command `mvn package` and quarkus will generate the output in the **/target** folder. For the native executables, the command needs more parameters but this is explained in the link above.
+
+Configure the Output with these https://quarkus.io/guides/maven-tooling#configuration-reference[configuration properties]
+
+
+== Create and build a Docker Image
+
+Quarkus supports Jib, S2I and https://www.docker.com/[Docker] for building images. We focus on building a Quarkus App with Docker.
+You get a created Dockerfile from Quarkus in the src/main/docker folder of any created project. There are multiple Dockerfiles.
+
+.Dockerfiles
+. Dockerfile.jvm + 
+Dockerfile for Quarkus application in the JVM mode. running in **Red Hat Universal Base Image 8 Minimal Container**
+. Dockerfile.legacy-jar + 
+DockerFile for Quarkus application in JVM mode with the legacy jar **running in Red Hat Universal Base Image 8 Minimal Container**.
+. Dockerfile.native + 
+Dockerfile using the native executable running in **Red Hat Universal Base Image 8 Minimal container**.
+. Dockerfile.native-distroless
+The native file will run in a **Distroless container**. Distroless images are very small containers with just the application and runtime dependencies and without the other programs coming with a Linux distribution.
+
+[Note]
+====
+For more information to the different executables go back to the chapter <<Packaging of a Quarkus application and creation of a native Executable>> 
+====
+
+To perform Docker builds with the generated Dockerfiles from above you need to add the following extension to your project with the command `mvn quarkus:add-extension -Dextensions="container-image-docker`. 
```

Comment:
I think you need to differentiate this part more and explain that this extension is not necessarily required to create the Docker image. Someone can simply create the image using the Dockerfiles and the commands mentioned in the section below. And when using the extension, the building of the image is done directly when packaging the application. This was not that clear for me when reading this section

Reply:
That's right, I adjusted the structure of the section and added a sentence to make it clearer.